pkg/binlog/reader/tcp_test.go

```diff
@@ -171,55 +95,11 @@ func (t *testTCPReaderSuite) TestSyncPos(c *C) {
 	// already closed
 	err = r.Close()
 	c.Assert(err, NotNil)
-
-	// invalid startup position
-	pos = gmysql.Position{
-		Name: "mysql-bin.888888",
```

Comment:
delete some tests lines beacause we just test wheather `TCPreader` status is  right, some test about  binlog events should be tested in it test